test/method_detection.test.js

```diff
@@ -84,7 +84,7 @@ test('test lodash-CC-style function detection', function (t) {
     function baseMerge() {}
   });
   var _ = runInContext();
-})();
+}.call(this));
```

Comment:
doesn't this remove the test for the previous form, without `call(this)`?

Reply:
It does. We haven't seen that in the wild, so I didn't duplicate the test. Can you think of a better way than just copy-pasting the whole test?